README.md

```diff
@@ -1,35 +1,67 @@
+<a name="readme-top"></a>
+
+
+<!-- PROJECT SHIELDS -->
+<!--
+  ** Shields utilize markdown "reference style" links for readability.
+  ** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+  ** See the bottom of this document for the declaration of the reference variables.
+  ** https://www.markdownguide.org/basic-syntax/#reference-style-links
+  ** -->
 <div align="center">
-  <img src="https://github.com/twosixlabs/armory/blob/master/tools/static_content/logo.png" width="50%" title="ARMORY logo">
+
+[![CI][ci-badge]][ci-url]
```

Comment:
CI badge appears broken at https://github.com/christopherwoodall/armory/tree/pre-release-cleanup

Reply:
My CI badge isn't working, but the hyperlink is using the "relative" repository prefix `/../../`. This will cause all URLs to redirect to the `main` branch page; e.g. in my fork `/../../` == `https://github.com/christopher/armory`, but in the ORG repo `/../../` == `https://github.com/twosixlabs/armory`.